deployment-templates/compute-engine/forseti-instance.py

```diff
@@ -205,8 +207,8 @@ def GenerateConfig(context):
 )"
 echo "$FORSETI_ENV" > $USER_HOME/forseti_env.sh
 
-sudo su $USER -c "$FORSETI_HOME/scripts/gcp_setup/bash_scripts/run_forseti.sh"
-(echo "{run_frequency} $FORSETI_HOME/scripts/gcp_setup/bash_scripts/run_forseti.sh") | crontab -u $USER -
+sudo su $USER -c "$FORSETI_HOME/setup/installer/scripts/forseti_run.sh"
```

Comment:
Is there a reason to change ```run_forseti.sh``` to ```forseti_run.sh```.  It was actually pretty nice to read before.  If anything, this change should be to a noun, like ```forseti_runner.sh```. 

This also has been done elsewhere, and the reading of those files seems awkward.

Reply:
To me it made more sense (especially in one directory) to organize them based on the object then the verb.

Reply:
Then for organization purposes, we should go with all noun as with (most) of our module namings.  The object/verb style is really unnatural in English.  So, being easy to read & say will be super-awesome and help us so much down the road.

 ```forseti_runner.sh```.

Similiarly, use
```docker_builder.sh```
```docker_runner.sh```
```forseti_services_initializer.sh```

Reply:
Sure, this should mostly be the case. I'm happy to switch conventions if we think it's best.